test/distributed/test_symmetric_memory.py

```diff
@@ -196,6 +196,9 @@ def test_get_signal_pad(self) -> None:
         symm_mem = _SymmetricMemory.rendezvous(t)
         peer_rank = (self.rank + 1) % self.world_size
 
+        signal_pad = symm_mem.get_signal_pad(self.rank)
```

Comment:
Some suggestions on making this test more robust:
- allocate a buffer of size zero, and then get its signal pad and write something to it (this is how I found this bug)
- write a signal to the signal pad, fill the buffer with zeros, check that the signal is still the same

Reply:
Appreciate the suggestion! Incorporated it into the test.